plugins/embark-slither/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "embark-slither",
+  "version": "4.0.0-beta.0",
+  "author": "Iuri Matias <iuri.matias@gmail.com>",
+  "contributors": [],
+  "description": "Embark Slither plugin",
+  "homepage": "https://embark.status.im/",
+  "keywords": [
+    "blockchain",
+    "dapps",
+    "ethereum",
+    "serverless",
+    "solc",
+    "solidity",
+    "slither"
+  ],
+  "license": "MIT",
+  "repository": "github:embark-framework/embark",
+  "main": "./dist/index.js",
+  "scripts": {
+    "build": "cross-env BABEL_ENV=node babel src --copy-files --extensions \".js,.ts\" --out-dir dist --source-maps",
+    "ci": "npm run qa",
```

Comment:
This is most likely my limited knowledge of lerna, but wouldn't this not be needed in a package? Shouldn't the top-level `ci` script take care of building all packages? Possibly the same for `lint`, `clean`, `watch`, `typecheck`, etc

Reply:
That will be removed as extracted into his own repo